transport-native-epoll/src/test/java/io/netty/channel/epoll/EpollSocketChannelTest.java

```diff
@@ -355,12 +380,12 @@ public void channelReadComplete(ChannelHandlerContext ctx) throws Exception {
 
         void assertSingleRead() throws InterruptedException {
             assertTrue(latch.await(5, TimeUnit.SECONDS));
-            assertEquals(1, count.get());
+            assertTrue("count: " + count.get(), count.get() >= 1);
```

Comment:
my point is either remove the message completely or make it complete ;) Whatever you like...